packages/emoji-mart/src/components/Picker/Picker.js

```diff
@@ -334,7 +334,8 @@ export default class Picker extends Component {
   }
 
   handleSearchInput = async () => {
-    const input = this.refs.searchInput.current
+    const input =
+      this.props.searchInputRef?.current ?? this.refs.searchInput.current
```

Comment:
Would be safer to have a `this.getSearchInputRef` method to avoid having to duplicate this logic every time we need to use the search input ref.